cqrs-spring-utils/build.gradle

```
@@ -0,0 +1,6 @@
+dependencies {
+
+    implementation project(":bus-cqrs-api")
```

Should be `api`, the types from `:bus-cqrs-api` are exposed to consumers of this module right?
```suggestion
    api project(":bus-cqrs-api")
```